pages/requester/create-task.tsx

```diff
@@ -295,7 +394,7 @@ export default function Form() {
               <Grid item xs={6}>
                 <Box display="flex" alignItems="center" justifyContent="left">
                   <Typography color="primary" sx={{ textAlign: "right" }}>
-                    Total ETH allocated:
+                    Total MATIC allocated:
```

Comment:
I would do a global search for "ETH" and replace as many places as possible with "MATIC"